community/kernel/src/main/java/org/neo4j/kernel/impl/factory/GraphDatabaseFacadeFactory.java

```diff
@@ -93,6 +94,8 @@
                 setting( "dbms.tracer", Settings.STRING, (String) null ); // 'null' default.
 
         public static final Setting<String> editionName = setting( "edition", Settings.STRING, "Community" );
+        public static final Setting<UsageDataKeys.OperationalMode> operationalMode = setting( "operationalMode",
```

Comment:
I think this should not be a setting because it is more like a metadata and it should not be overwritable.
Instead all interested parties should probably be able to access `OperationalMode` from `KernelContext`.